.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:8-browsers
+      # - image: circleci/node:6-browsers
+        environment:
+            CHROME_BIN: "/usr/bin/google-chrome"
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/mongo:3.4.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
+          name: Install bower
+          command: 'sudo npm install -g bower'
+
+      - run: yarn install
+      - run: bower install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+
+      # connect
+      - run: yarn browserstack-connect
+
+      # run tests!
+      - run: yarn test
+
+      # deploy
+      - run:
+          name: Avoid hosts unknown for github
+          command: ssh-keyscan planninglabs.nyc >> ~/.ssh/known_hosts
+      - deploy:
+          name: Deployment
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              git remote add dokku dokku@planninglabs.nyc:map-skeleton-app && git push --force dokku master
+            elif [ "${CIRCLE_BRANCH}" == "develop" ]; then
+              git remote add dokku dokku@planninglabs.nyc:map-skeleton-app && git push --force dokku develop:master
```

Comment:
This should add the staging version:
git remote add dokku dokku@planninglabs.nyc:map-skeleton-app-staging && git push --force dokku develop:master

Reply:
Changed ! :)